I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 10, in script call
    call splash
  File "game/combatsystem.rpy", line 82, in script
    show versus with moveindown
  File "game/combatsystem.rpy", line 82, in <module>
    show versus with moveindown
NameError: name 'moveindown' is not defined

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 10, in script call
    call splash
  File "game/combatsystem.rpy", line 82, in script
    show versus with moveindown
  File "C:\Users\svlaw\Downloads\renpy-6.99.12.3-sdk\renpy\ast.py", line 1271, in execute
    paired = renpy.python.py_eval(self.paired)
  File "C:\Users\svlaw\Downloads\renpy-6.99.12.3-sdk\renpy\python.py", line 1745, in py_eval
    return py_eval_bytecode(code, globals, locals)
  File "C:\Users\svlaw\Downloads\renpy-6.99.12.3-sdk\renpy\python.py", line 1739, in py_eval_bytecode
    return eval(bytecode, globals, locals)
  File "game/combatsystem.rpy", line 82, in <module>
    show versus with moveindown
NameError: name 'moveindown' is not defined

Windows-8-6.2.9200
Ren'Py 6.99.12.3.2123
BattleTest 1.0
